test/dshell/dwarf.d

```diff
@@ -0,0 +1,83 @@
+import dshell;
+
+import std.algorithm;
+import std.file;
+import std.process;
+
+int main()
+{
+    version(DigitalMars) { }
+    else
+    {
+        writeln("Skipping dwarf.d for non-DMD compilers.");
+        return DISABLED;
+    }
+
+    version(Windows)
+    {
+        writeln("Skipping dwarf.d for Windows.");
+        return DISABLED;
+    }
+
+
+    // If the Unix system doesn't have objdump, disable the tests
+    auto sysHasObjdump = executeShell("objdump --help");
+    if (sysHasObjdump.status)
+        return DISABLED;
+
+    bool failed;
+
+    // test them all
+    foreach (string path; dirEntries(EXTRA_FILES ~ "/dwarf/", SpanMode.shallow))
```

Comment:
buildPath as this might be run on Windows at some point.

Reply:
On Windows, both the backslash and the slash are in principle valid directory separators.
Do I really need to change every slash to a backslash, even if this test is disabled, on Windows ?

Reply:
In theory yes, but better safe than sorry as Windows isn't really reliable in that regard.

(You'll need to push anyway to restart SemaphoreCI)